pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -196,6 +201,19 @@ type PermissionClaim struct {
 	IdentityHash string `json:"identityHash,omitempty"`
 }
 
+type ClaimedResource struct {
+	// name of an object within a claimed group/resource.
+	// It matches the metadata.name field of the underlying object.
+	// If unset, all objects having the group/resource are being claimed.
+	// +optional
+	Name string `json:"name,omitempty"`
+
+	// namespace containing the named object. Matches metadata.namespace field.
+	// If namespace is set but name is not set this field is ignored.
```

Comment:
this is strange. How can I claim a complete namespace?

Reply:
As I understand it, it's for scoping named resources to a namespace.

I suppose one could try to claim all instances of the RG in a namespace, but so far I haven't thought of or seen a use case for that.

Reply:
@sttts it was a suggestion from your side when we did a quick chat. We can remove it if it doesn't make sense though.